gallery/experiments/README.rst

```diff
@@ -1,4 +1,5 @@
 Experiments
 =============================
 
-This gallery will be for demonstrating the functionality of ASPIRE tools using experimental data. 
+This gallery contains example experiments for 3-D ab-initio modeling using experimental data as well as
+data simulated by ASPIRE's homegrown methods for generating realistic Cryo-EM data.
```

Comment:
Kind of liked the old phrasing here avoiding the repetition of "experiment"

This gallery contains examples demonstrating the functionality of ASPIRE tools for 3-D ab-initio reconstruction. Experimental cryo-EM data and synthetic data created by ASPIRE's `Simulation` class are used.

Reply:
I like it. I'll update. Thanks!